float32 lidar_alt
bool switch_status